src/drivers/config/drivers.yaml

```diff
@@ -28,4 +28,4 @@ version: "384.111"
 pre-installed-nvidia-path: /usr/local/nvidia
 
 
-skip-ib-installation: false
+skip-ib-installation: true
```

Comment:
1. Can the name of configuration is positive (enable-IB-driver-installation), instead of negative(skip, disable, turn-off)?
2. add some comments and update document to explain this setting, and potential issue once it's enabled.

Reply:
updated